@xen-orchestra/fs/src/abstract.js

```diff
@@ -440,6 +440,14 @@ export default class RemoteHandlerAbstract {
   async _unlink(file: string): Promise<void> {
     throw new Error('Not implemented')
   }
+
+  async getInfoDisk(dir: string): Promise<number> {
```

Comment:
Should be named something like `getInfo` and should not accept a path but simply returns info on the whole remote.